test/common/upstream/health_check_fuzz.proto

```diff
@@ -95,4 +95,5 @@ message HealthCheckTestCase {
   repeated Action actions = 2;
   bool http_verify_cluster = 3; //Determines if verify cluster setting is on
   bool start_failed = 4;
+  bool upstream_cx_total_inc = 5;
```

Comment:
This is a really confusing title. Can you change this to "upstream_cx_success", as in, upstream connection success?

Reply:
Changed.